proto/user.proto

```diff
@@ -78,6 +78,15 @@ message DropUserResponse {
   uint64 version = 2;
 }
 
+message UpdateUserRequest {
+  UserInfo user = 1;
```

Comment:
Altering from multi-frontends might cause inconsistency. We should provide some mechanism like self-defined `FieldMask` as follow:
```proto
message UpdateUserRequest {
  enum UpdateField {
    UNKNOWN = 0;
    SUPER = 1;
    LOGIN = 2;
    CREATE_DB = 3;
    AUTH_INFO = 4;
  }
  UserInfo user = 1;
  repeated UpdateField update_fields = 2;
}
```